src/utils/featheryClient/index.ts

```diff
@@ -994,4 +994,39 @@ export default class FeatheryClient extends IntegrationClient {
       }
     });
   }
+
+  async generateDocuments({
+    documentIds,
+    downloadEnvelopes = false
+  }: {
+    documentIds: string[];
+    downloadEnvelopes?: boolean;
+  }) {
+    const { userId } = initInfo();
+    const payload: Record<string, any> = {
+      form_key: this.formKey,
+      fuser_key: userId,
+      documents: documentIds
+    };
+
+    const url = `${API_URL}document/form/generate/`;
+    return this._fetch(
+      url,
+      {
+        headers: { 'Content-Type': 'application/json' },
+        method: 'POST',
+        body: JSON.stringify(payload)
+      },
+      false
+    ).then(async (response) => {
+      if (response) {
+        if (response.ok) {
+          const data = await response.json();
+          const files = data.files;
+          if (downloadEnvelopes) await downloadAllFileUrls(files);
+          return files;
```

Comment:
let's return this in a format like { files } so in the future if we want to add other parameters to return it'll be easy to do it in a backwards compatible way